Gemfile

```diff
@@ -13,9 +13,9 @@ gem "i18n", "~> 1.8.1"
 
 gem "decidim", DECIDIM_VERSION
 gem "decidim-consultations", DECIDIM_VERSION
-gem "decidim-decidim_awesome", "~> 0.7.2"
-gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer", branch: "0.24-stable"
-gem "decidim-verifications-custom_csv_census", git: "https://github.com/CodiTramuntana/decidim-verifications-custom_csv_census.git", tag: "v0.0.2"
+gem "decidim-decidim_awesome", "~> 0.8"
+gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer", branch: "release/0.25-stable"
+gem "decidim-verifications-custom_csv_census", git: "https://github.com/Platoniq/decidim-verifications-custom_csv_census", branch: "chore/upgrade-0.25"
```

Comment:
@apuertadelaguila please change this line to use the branch: `release/0.25-stable` and it'll be ready